package.json

```diff
@@ -21,22 +21,30 @@
   },
   "homepage": "https://github.com/Visual-Testing-for-Android-Apps/web#readme",
   "devDependencies": {
+    "css-loader": "^5.2.1",
```

Comment:
Correct me if I'm wrong, but since we are building on the server, these will need to be regular dependencies